sql/core/src/main/scala/org/apache/spark/sql/execution/SortExec.scala

```diff
@@ -206,3 +179,43 @@ case class SortExec(
   override protected def withNewChildInternal(newChild: SparkPlan): SortExec =
     copy(child = newChild)
 }
+object SortExec {
+  def createSorter(
```

Comment:
this change is because `maxConcurrentOutputFileWriters` need to create sorter at `FileFormatWriter`